config/application.rb

```diff
@@ -8,6 +8,9 @@
 
 module Awbw
   class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 5.0
```

Comment:
Should this be `5.2`?

Reply:
This needs to be set to the originally generated Rails version.  In our case it's really older than this, but this setting only began to be available starting with Rails 5.0.

We can move this forwards eventually, but at this point hoping to just stand everything up and getting it working such that this understands:
* the Rails version where we had generally originated, and
* with the code really using the Rails version that we'd like to upgrade to -- in this case 5.2.8.1.